docs/man/xrdp.ini.5.in

```diff
@@ -23,6 +23,25 @@ All options and values (except for file names and paths) are case insensitive, a
 .SH "GLOBALS"
 The options to be specified in the \fB[Globals]\fR section are the following:
 
+.TP
+\fBport\fR=\fIport_specification [ port_specification ... ]\fR
+Specify the port(s) that xrdp should listen on. More instructions and
+examples can be found within xrdp.ini itself.
+
+.TP
+\fBvmconnect\fR=\fI[true|false]\fR
+(Hyper-V VMs only). Allows the user to skip some unnecessary security
+checks when a virtual machine running xrdp is hosted on Hyper-V,
+and the user connects to it via the vmms service on TCP port 2179.
```

Comment:
I find the "unnecessary checks" part a bit confusing. The rest is good.
Here's a suggestion that just omits it:

```suggestion
(Hyper-V VMs only). Enables a wider support of security protocols when a virtual machine running xrdp is hosted on Hyper-V,
and the user connects to it via the vmms service on TCP port 2179.
```

Reply:
Agreed.